python/doc/examples/functional_modeling/functional_basis/plot_multidimensional_basis.py

```diff
@@ -0,0 +1,59 @@
+"""
+============================================================================
+Create a basis of functions with input and output dimension greater than one
+============================================================================
+"""
+# %%
+# Import the required modules
+
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a basis :math:`\mathcal{B}=\{f_1,...,f_n\}` such that :math:`f_n: \mathbb{R}^d \mapsto \mathbb{R}^q` with :math:`d>1, q>1`.
+#
+# We can do that:
+#
+# - Case 1: by the aggregation of :math:`n` given functions :math:`f_i: \mathbb{R}^d \mapsto \mathbb{R}^q`,
+# - Case 2: by the aggregation of :math:`n` functions :math:`f_i: \mathbb{R}^d \mapsto \mathbb{R}^q` built from factories of functions :math:`g_j:\mathbb{R}^d \mapsto \mathbb{R}`.
+#
+
+# %%
+# Case 1: Aggregation of given functions
+# ======================================
+#
```

Comment:
At this point, a comment would clarify the meaning of the example. I already know what is a `AggregatedFunction`, but the way this class articulates your example is implicit here.

Reply:
ok I am going to see.